src/containerapp/azext_containerapp/java_component_decorator.py

```diff
@@ -57,6 +57,10 @@ def create(self):
                 environment_name=self.get_argument_environment_name(), name=self.get_argument_java_component_name(),
                 java_component_envelope=self.java_component_def, no_wait=self.get_argument_no_wait())
         except Exception as e:
+            stringErr = str(e)
+            if "JavaComponentsNotAllowedForSubscription" in stringErr:
```

Comment:
Is it possible to get ErrorCode?

Reply:
The error code is 400. You mean that we need to check both of the error code and the error message, right?

Reply:
As we discussed offline, will close this